### PR TITLE
Chef Infra Server Updates

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
### :nut_and_bolt: Description: What code changed, and why?

- Policyfiles tab under Policy Group
- Nodes tab under Policy Group
- Pagination in Policy Group
- Slider added for the specific Cookbooks
- Runlist tab added for the Policyfiles section

### :chains: Related Resources

- https://github.com/chef/automate/pull/5498
- https://github.com/chef/automate/pull/5630
- https://github.com/chef/automate/pull/5640
- https://github.com/chef/automate/pull/5553
- https://github.com/chef/automate/pull/5619

### :+1: Definition of Done

All the above changes have been made to the Chef Infra Server Page

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -489,7 +504,12 @@ You can **Expand** and **Collapse** the *default* and *override* attributes by s
 
 ##### Cookbook Dependencies
 
-Select the **Cookbook Dependencies** button to view all the *Dependencies Rules* and *Cookbook* of the policyfile in table format. The slider tab also contains the `Operator` and `Version` of the cookbooks.
+Select the **Cookbook Dependencies** button to view all the *Dependencies Rules* and *Cookbook* of the policyfile in table format. Select a specific cookbook from the list to view the details of the cookbook.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:507]: If these should be italics, then please use the _ (underscore). Yes, I know that both work, but the underscore lets us know your intent.

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Small changes & we're diagnosing the build failures.

**Comment by kagarmoe** [comment]
💥 Fixed the netlify problem.

**Review by himanshi-chhabra — APPROVED** [review]

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 2]
```diff
@@ -473,12 +473,27 @@ Select a Policyfile to view the **METADATA**, **Content**, **Attributes**, **Rev
 
 ##### Content Tab
 
-The **Content** tab contains the list of all the *Included Policies* and *Run Lists*. Select a specific Policyfile from the *Included Policies* to view the details.
+The **Content** tab contains the list of all:
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:476]:
```suggestion
The **Content** tab lists the:
```

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 3]
```diff
@@ -473,12 +473,27 @@ Select a Policyfile to view the **METADATA**, **Content**, **Attributes**, **Rev
...
+
+- Included Policies
+- Run Lists
+
+**Included Policies**
+
+Select a specific Policyfile from the _Included Policies_ to view the details.
```
Comment (kagarmoe) [inline components/docs-chef-io/content/automate/chef_infra_server.md:483]:
```suggestion
Select a Policyfile from the _Included Policies_ to view the details.
```

**Review by kagarmoe — APPROVED** [review]
tenor-195940184

**Pull request merged**